ci/pipeline-haproxy-maintenance.yml

```diff
@@ -449,7 +449,7 @@ resources:
   - name: blobstore
     type: gcs
     source:
-      bucket:   haproxy-boshrelease
+      bucket:   haproxy-boshrelease-maintenance
```

Comment:
It should use the same bucket. The `haproxy-boshrelease-maintenance` bucket was only created for testing to not interfere with the live environment but should be deleted afterwards.

Reply:
okay but for semver version we are still using `haproxy-boshrelease-maintenance` https://github.com/cloudfoundry/haproxy-boshrelease/blob/d1aac0c8b3b6bd924dc3a03e4642ddcc5c71df6f/ci/pipeline-haproxy-maintenance.yml#L432 here I guess it should be like this?
and I see there is a config file here: https://github.com/cloudfoundry/haproxy-boshrelease/blob/064285b35ec133d1fea60dc30aa5bf7baee6ac58/config/final.yml#L6 here we should use haproxy-boshrelease-maintenance? as it is called in `shipit` script here: https://github.com/cloudfoundry/haproxy-boshrelease/blob/064285b35ec133d1fea60dc30aa5bf7baee6ac58/ci/scripts/shipit#L49

Reply:
That should have been adapted.

Reply:
I have change the bucket back to use `haproxy-boshrelease`. Will cross check version file with @b1tamara next week.